app/main.py

```diff
@@ -1,8 +1,66 @@
+from typing import List
+
+
 class Car:
-    # write your code here
-    pass
+    def __init__(self, comfort_class: int,
+                 clean_mark: int,
+                 brand: str) -> None:
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(self, distance_from_city_center: float, clean_power: int,
+                 average_rating: float, count_of_ratings: int) -> None:
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = average_rating
+        self.count_of_ratings = count_of_ratings
+
+    def serve_cars(self, cars: List[Car]) -> float:
+        total_income = 0.0
+        for car in cars:
+            if car.clean_mark < self.clean_power:
+                wash_cost = self.calculate_washing_price(car)
+                total_income += wash_cost
+                self.wash_single_car(car)
+        return round(total_income, 1)
+
+    def calculate_washing_price(self, car: Car) -> float:
+        return round(car.comfort_class * (self.clean_power - car.clean_mark)
+                     * self.average_rating / self.distance_from_city_center, 1)
+
+    def wash_single_car(self, car: Car) -> None:
+        if self.clean_power > car.clean_mark:
+            car.clean_mark = self.clean_power
+
+    def rate_service(self, rating: float) -> None:
+        self.average_rating = round(
+            (self.average_rating * self.count_of_ratings + rating)
+            / (self.count_of_ratings + 1), 1)
+        self.count_of_ratings += 1
+
+
+# Приклад використання
+bmw = Car(3, 3, "BMW")
+audi = Car(4, 9, "Audi")
+mercedes = Car(7, 1, "Mercedes")
+
+ws = CarWashStation(6, 8, 3.9, 11)
+
+income = ws.serve_cars([bmw, audi, mercedes])
+print(income == 41.7)
+
+print(bmw.clean_mark == 8)
+print(audi.clean_mark == 9)
+print(mercedes.clean_mark == 8)
+
+ford = Car(2, 1, "Ford")
+wash_cost = ws.calculate_washing_price(ford)
+print(wash_cost == 9.1)
+print(ford.clean_mark == 1)
+
+ws.rate_service(5)
+print(ws.count_of_ratings == 12)
+print(ws.average_rating == 4.0)
```

Comment:
Check the checklist.md ending for this task, 

> Add comments, prints, and functions to check your solution when you write your code. Don't forget to delete them when you are ready to commit and push your code.
**So, there is no need to leave tests and comments here 😊**